src/current/v23.2/admission-control.md

```diff
@@ -24,21 +38,39 @@ Admission control can help if your cluster has degraded performance due to the f
 
+## Operations subject to Admission Control
+
+Almost all database operations that use CPU or perform Storage IO are controlled by the admission control system. From a user's perspective, specific operations that are affected by admission control include:
+
+- [General SQL queries]({% link {{ page.version.version }}/selection-queries.md %}) have their CPU usage subject to admission control, as well as storage IO for writes to [leaseholder replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases).
+- [Bulk data imports]({% link {{ page.version.version }}/import-into.md %}).
+- [Backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}).
+- [Schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), including index and column backfills (on both the [leaseholder replica]({% link {{ page.version.version }}/architecture/replication-layer.md %}#leases) and [follower replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft)).
+- [Deletes]({% link {{ page.version.version }}/delete-data.md %}) (including deletes initiated by [row-level TTL jobs]({% link {{ page.version.version }}/row-level-ttl.md %}); the [selection queries]({% link {{ page.version.version }}/selection-queries.md %}) performed by TTL jobs are also subject to CPU admission control).
+- [Follower replication work]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft).
+- [Raft log entries being written to disk]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft).
+- [Changefeeds]({% link {{ page.version.version }}/create-and-configure-changefeeds.md %}).
+- [Intent resolution]({% link {{ page.version.version }}/architecture/transaction-layer.md %}#write-intents).
+
+The following operations are not subject to admission control:
+
+- SQL writes are not subject to admission control on [follower replicas]({% link {{ page.version.version }}/architecture/replication-layer.md %}#raft) by default, unless those writes occur in transactions that are subject to a Quality of Service (QoS) level as described in [Set quality of service level for a session](#set-quality-of-service-level-for-a-session). In order for writes on follower replicas to be subject to admission control, the setting `default_transaction_quality_of_service=background` must be used.
+
 {{site.data.alerts.callout_info}}
 Admission control is beneficial when overall cluster health is good but some nodes are experiencing overload. If you see these overload scenarios on many nodes in the cluster, that typically means the cluster needs more resources.
 {{site.data.alerts.end}}
 
 ## Enable and disable admission control
 
-To enable and disable admission control, use the following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}):
+Admission control is enabled by default.
+
```

Comment:
Extra line break here.

Reply:
ya I added that for emphasis but I'm not wedded to it.  Removed